recipe/build_mamba.bat

```diff
@@ -36,5 +36,8 @@ if /I "%PKG_NAME%" == "mamba" (
     cmake --build build-mamba/ --parallel %CPU_COUNT%
     if errorlevel 1 exit 1
     cmake --install build-mamba/
-
+    :: Place mamba executable copy to condabin/
+    mkdir "%PREFIX%\condabin"
+    echo ^"%LIBRARY_BIN%\mamba.exe^" %%* > "%PREFIX%\condabin\mamba.bat"
```

Comment:
I need to check but I'm pretty sure this would break the mamba acivate / mamba deactivate features.

Reply:
We can also place a copy of the full executable there. Would that break activation?